packages/agoric-cli/test/inter-cli.test.js

```diff
@@ -252,7 +252,7 @@ test('amount parsing', t => {
   });
 });
 
-test.todo('want as max collateral wanted');
+// test.todo('want as max collateral wanted');  see https://github.com/Agoric/agoric-sdk/issues/9965
```

Comment:
if it's commented out, there's no point in using `test.todo`.
```suggestion
// TODO want as max collateral wanted'  see https://github.com/Agoric/agoric-sdk/issues/9965
```

but instead I'd suggest removing replacing these multiple lines with one:
```
// TODO improve test coverage https://github.com/Agoric/agoric-sdk/issues/9965
```